sbin/build.sh

```diff
@@ -408,47 +422,49 @@ printJavaVersionString()
 }
 
 # Clean up
-removingUnnecessaryFiles()
-{
+removingUnnecessaryFiles() {
+  local openJdkVersion="$1"
+  local jdkTargetPath="jdk-$1"
+  local jreTargetPath="jre-$1"
+
   echo "Removing unnecessary files now..."
 
-  local openJdkVersion=$(getOpenJdkVersion)
   stepIntoTheWorkingDirectory
 
   cd build/*/images || return
 
   echo "Currently at '${PWD}'"
 
+
   local jdkPath=$(ls -d ${BUILD_CONFIG[JDK_PATH]})
-  echo "moving ${jdkPath} to ${openJdkVersion}"
-  rm -rf "${openJdkVersion}" || true
-  mv "${jdkPath}" "${openJdkVersion}"
+  echo "moving ${jdkPath} to ${jdkTargetPath}"
+  rm -rf "${jdkTargetPath}" || true
+  mv "${jdkPath}" "${jdkTargetPath}"
 
   if [ -d "$(ls -d ${BUILD_CONFIG[JRE_PATH]})" ]
   then
-    JRE_TARGET_PATH="${openJdkVersion}-jre"
-    [ "${JRE_TARGET_PATH}" == "${openJdkVersion}" ] && JRE_TARGET_PATH="${openJdkVersion}.jre"
-    echo "moving $(ls -d ${BUILD_CONFIG[JRE_PATH]}) to ${JRE_TARGET_PATH}"
-    rm -rf "${JRE_TARGET_PATH}" || true
-    mv "$(ls -d ${BUILD_CONFIG[JRE_PATH]})" "${JRE_TARGET_PATH}"
+    [ "${jreTargetPath}" == "${openJdkVersion}" ] && jreTargetPath="${openJdkVersion}.jre"
```

Comment:
Should that be `jreTargetPath=="${openJdkVersion}.jre"` I'mconfused why an assignment occurs after an `&&`